### DISCUSSION TITLE
Can't get RPGLE Outline to Display Again

### DISCUSSION THREAD

**Discussion by brianrocket13** [codefori/vscode-ibmi#2340] [Q&A] [answered]
When this happened last year, it was because I didn't have the RPG Language Tools installed. Well, this time it is installed on a new PC and the outline is not being displayed. I even went in and uninstalled all of the extensions and then reinstalled Code for IBM i which caused Code for IBM i Walkthroughs and Syntax Highlighting for IBM i Languages to be installed and I reinstalled RPG Language Tools and it still didn't work.  I used very small simple source to test it.  I tried multiple Source Members. 

**FREE                          
  Dcl-S ToDate Zoned(8:0);      
  Dcl-S WorkDate Date;          
                                
  ToDate = 20241109;            
  WorkDate = %DATE(ToDate:*ISO);
  Dsply ToDate;                 
  Dsply WorkDate;               
  Dsply (WorkDate + %MONTHS(1));
  *INLR = *ON;

**Comment by worksofliam** [comment, marked as answer]
Trying searching 'Focus Outline View' in the Command Palette (F1)

<img width="664" alt="image" src="https://github.com/user-attachments/assets/326f1c07-91ef-455c-bd36-a18bdf66f993">

**Reply by brianrocket13** [reply]
That fixed it. Thank you very much!
